flow/include/flow/flow.h

```diff
@@ -100,6 +100,8 @@ extern StringRef strinc(StringRef const& str, Arena& arena);
 extern Standalone<StringRef> addVersionStampAtEnd(StringRef const& str);
 extern StringRef addVersionStampAtEnd(StringRef const& str, Arena& arena);
 
+int nChooseK(int n, int k);
```

Comment:
A comment explaining this function would be nice.

Reply:
will do